.github/workflows/root-ci-config/build_root.py

```diff
@@ -348,6 +363,17 @@ def build(options, buildtype):
 
     cmake_build(buildtype)
 
+
+@github_log_group("Create binary packages")
+def create_binaries(buildtype):
```

Comment:
Why is this function needed only now?

Reply:
The new CI never had the ability to create binary packages. This PR introduces this feature.